lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1866,6 +1886,22 @@ def api_client
       @client
     end
 
+    def use_secure_channel
```

Comment:
I think having this as a separate method detracts from readability. Would rather see this test inline.

Reply:
Inlined.